Error reported by phpstan 0.12.92 which causes CI fail

```
$ make test-php-phpstan
phpstan composer.lock is not up to date.
composer bin phpstan install --no-progress
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking phpstan/phpstan (0.12.92)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing phpstan/phpstan (0.12.92): Extracting archive
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ --------------------------------- 
  Line   lib/KeyManager.php               
 ------ --------------------------------- 
  432    Left side of && is always true.  
 ------ --------------------------------- 

                                                                                                                        
 [ERROR] Found 1 error                                                                                                  
                                                                                                                        

make: *** [Makefile:112: test-php-phpstan] Error 1

```

--- a/lib/KeyManager.php
+++ b/lib/KeyManager.php
@@ -429,7 +429,7 @@ class KeyManager {
 			$privateKey = $this->session->getPrivateKey();
 		}
 
-		if ($encryptedFileKey && $shareKey && $privateKey) {
+		if ($shareKey && $privateKey) {
 			return $this->crypt->multiKeyDecrypt(
 				$encryptedFileKey,
 				$shareKey,

